bu/CopcSupport.cpp

```diff
@@ -42,11 +42,15 @@ CopcSupport::CopcSupport(const BaseInfo& b) : m_b(b),
     m_f.open(toNative(b.opts.outputName), std::ios::out | std::ios::binary);
 
     //ABELL
```

Comment:
These "//ABELL" comments should be removed. There were just there to note that the values were crap, but you've fixed that.

Reply:
fixed